pybv/io.py

```diff
@@ -246,43 +245,41 @@ def _optimize_channel_unit(resolution, unit):
 
 
 def _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, data, sfreq, ch_names,
-                     orientation='multiplexed', format='binary_float32',
-                     resolution=1e-7, unit='µV'):
+                     orientation, format, resolution, unit):
     """Write BrainvVision header file."""
-    fmt = format.lower()
     bvfmt, _ = _chk_fmt(format)
 
     multiplexed = _chk_multiplexed(orientation)
 
     with codecs.open(vhdr_fname, 'w', encoding='utf-8') as fout:
         print(r'Brain Vision Data Exchange Header File Version 1.0', file=fout)
-        print(r';Written using pybv {}'.format(__version__), file=fout)
+        print(r'; Written using pybv {}'.format(__version__), file=fout)
```

Comment:
in the "real" bv files, there is always a space after the start of a comment - so do it here as well